src/fuse/file_system.cr

```diff
@@ -51,6 +51,11 @@ module Fuse
       nil
     end
 
+    # Reads a symlink.  Return 0 on success.
```

Comment:
You meant to write "Return `nil` on success" I guess

Reply:
ah, no. I decided to update the method signature but forgot about the comment. I'll fix that!